sdk/trts/trts_veh.cpp

```diff
@@ -229,7 +229,7 @@ extern "C" __attribute__((regparm(1))) void internal_handle_exception(sgx_except
         continue_execution(info);
     }
 
-    if ((nhead = (uintptr_t *)malloc(size)) == NULL)
+    if ((nhead = (uintptr_t *)alloca(size)) == NULL)
```

Comment:
I think some comments should be added here to highlight why `alloca` is used here.

Even more, it makes senses to add comments to remind ourselves that a user-registered exception handler may never come back, which may have unexpected consequences to the correctness and safety of the logic of `internal_handle_exception` function.

For example, I am dubious about the following code after the removed `free`:

```
    if(EXCEPTION_CONTINUE_EXECUTION == status)
    {
        //exception is handled, decrease the nested exception count
        thread_data->exception_flag--;
    }
    else
    {
        //exception cannot be handled
        thread_data->exception_flag = -1;
    }
```

This code snippet is also skipped. But is there any unexpected consequences, especially in terms of security?

Reply:
Updated the exception_flag part.